boa/environment.py

```diff
@@ -470,9 +478,9 @@ class FakeMessage(Message):  # Message object with settable attrs
         return self.vm.state.computation_class.apply_message(self.vm.state, msg, tx_ctx)
 
     # function to time travel
-    def time_travel(self, sleep_time):
-        self.vm.patch.timestamp += sleep_time
-        self.vm.patch.block_number += sleep_time // 12
+    def time_travel(self, time_delta: int, block_delta: int = 12):
```

Comment:
i think the implementation should be more like
```suggestion
    def time_travel(self, seconds = None, blocks = None):
        if seconds is None == blocks is None:
            raise ValueError("one of seconds or blocks should be set")
        if seconds is not None:
            blocks = seconds // 12
        else:
            seconds = blocks * 12
        self.vm.patch.timestamp += seconds
        self.vm.patch.block_number += blocks
```